src/test/shell/bazel/bazel_coverage_test.sh

```diff
@@ -64,6 +64,25 @@ int main(void) {
 EOF
 }
 
+# Returns 0 if gcov is not installed or if a version before 7.0 was found.
```

Comment:
I'd return 0 on success and error code instead

Reply:
or rename the function to is_wrong_gcov_installed :)

Reply:
Renamed it.